### ISSUE TITLE
`alternate_gh` in `dtcwt`

### ISSUE THREAD

**Issue by xir4n** [kymatio/murenn#63]
Experiments (below) show that alternating the filters couldn't result in an analytic filter. 
What I propose to do is to set `alternate_gh=False` as default in the `DTCWT` class.
```
from murenn import DTCWT, IDTCWT
import torch
from matplotlib import pyplot as plt
import numpy as np

def get_basis(j, **kwargs):
    N = 2**12
    x = torch.zeros(1, 1, N)

    fwd = DTCWT(**kwargs)
    inv = IDTCWT(**kwargs)

    lp, bpsr = fwd(x)
    _, bpsi = fwd(x)
    _, bpsc = fwd(x)


    bpsj = torch.zeros(bpsr[j].shape)
    bpsj[:,:,bpsj.shape[2]//2] = 1 / np.sqrt(2)**j

    bpsr[j] = bpsj * (1+0j)
    bpsi[j] = bpsj * (0+1j)


    yr = inv(lp, bpsr).detach()
    yi = inv(lp, bpsi).detach()

    return yr, yi

for j in range(8):
    yr, yi = get_basis(j, alternate_gh=False)
    yc = yr + 1j*yi
    yc_hat = np.fft.fft(yc.squeeze().numpy())
    plt.plot(np.abs(yc_hat)/np.abs(yc_hat).max())
plt.title("alternate_gh=False")

for j in range(8):
    yr, yi = get_basis(j, alternate_gh=True)
    yc = yr + 1j*yi
    yc_hat = np.fft.fft(yc.squeeze().numpy())
    plt.plot(np.abs(yc_hat)/np.abs(yc_hat).max())
plt.title("alternate_gh=True")
```

![image](https://github.com/user-attachments/assets/c4882cb9-dac3-4271-819a-d93037d1a279)
![image](https://github.com/user-attachments/assets/33789202-b395-4435-966b-d329bd423fae)

_Originally posted by @xir4n in https://github.com/kymatio/murenn/pull/62#discussion_r1865602544_

**Comment by lostanlen** [comment]
Thanks. I think that `alternate_gh=True` is missing one conjugation for every other scale.
Blue is left-side, Orange is right-sided, Green is left-sided, etc.

But the bigger issue is that the transform does not do a good job of maintaining Hilbert analyticity, particularly at large scales. So this is the safest option.

**Comment by lostanlen** [comment]
Closed by #62
